src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp

```diff
@@ -411,31 +399,32 @@ void LoopEndEmitter::emit_code(const std::vector<size_t> &in,
 
 void LoopEndEmitter::validate_arguments(const std::vector<size_t> &in,
                                        const std::vector<size_t> &out) const {
-    if (loop_begin->input_regs.size() != num_inputs)
-        IE_THROW() << "Invalid loop_begin->input_regs size: expected " << num_inputs << " got " << loop_begin->input_regs.size();
+    if (!loop_begin->input_regs.empty())
```

Comment:
Please remove this check, and also the assignment on L369 - it's an nGraph IR legacy.

Reply:
Done, thanks